services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: str) -> CinemaHall:
```

Comment:
The parameter `hall_seats_in_row` is typed as `str`, but it should be `int` to match the `seats_in_row` field in the `CinemaHall` model, which is an `IntegerField`.

Reply:
+